website/docs/datasource/logical_view.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Logical View
+language: en
+sidebar_label: Logical View
+pagination_label: Logical View
+toc_min_heading_level: 2
+toc_max_heading_level: 6
+pagination_prev: null
+pagination_next: null
+keywords:
+    - logical view
+draft: false
+last_update:
+    date: 02/28/2023
+---
+
+## Logical View
+- Logical view is a special view maintained only in KYLIN rather than a real view in hive. Once created, you can use it like a normal Hive view.
+- Logical view feature is turned off by default, you can set `kylin.source.ddl.logical-view.enabled = true` to turn this feature on.
+- All logical views are in the same database, you can set the database name by the following parameters, be careful not to have the same name as the normal Hive database:
+  - `kylin.source.ddl.logical-view.database = KYLIN_LOGICAL_VIEW`
```

Comment:
`kylin.source.ddl.logical-view.enabled = true`
`kylin.source.ddl.logical-view.database = KYLIN_LOGICAL_VIEW`
please remove the spaces on both sides of `=`